### DISCUSSION TITLE
Not knowing I sent APT funds to public key instead of the account, are the funds lost?

### DISCUSSION THREAD

**Discussion by iamladi** [aptos-labs/aptos-developer-discussions#131] [Questions] [answered]
I have sent by not knowing (expecting the public key to be also the address of the account) APT funds to public key. Are the funds lost? Is there a way to get the funds from the public key to the correct account?

**Comment by kshitijc1506** [comment, marked as answer]
If you sent funds using the public key instead of account address, you most likely **cannot** recover those funds. 

Both public keys and account addresses are 32-byte hexadecimals. If you sent funds to your account's public key, the Aptos blockchain will assume that it is an account address and will either:

1) Create a new account (where the account address is your public key). 
2) Send funds to the account address that already exists (where the account address is the same as your public key). In this context, this is a very unlikely scenario. 

To recover your funds, you would need access to the private key of that newly created account. 

If you're using a wallet like Petra to send funds to an account that doesn't exist, it will indicate that a new account will be created.
